int32 node_id
---
gvg_mapper/GVGEdgeMsg[] edges
